app/src/androidTest/java/org/kiwix/kiwixmobile/testutils/TestUtils.java

```diff
@@ -40,7 +40,7 @@
 import java.util.Date;
 import org.hamcrest.Description;
 import org.hamcrest.Matcher;
-import org.kiwix.kiwixmobile.core.entity.LibraryNetworkEntity.Book;
```

Comment:
You can undo this. Instead of writing this `LibraryNetworkEntity.Book book` You can write this `Book book`. It's a more general way to write code.